releasenotes/config.yaml

```diff
@@ -1,3 +1,49 @@
 ---
 encoding: utf8
 default_branch: main
+collapse_pre_releases: true
```

Comment:
We might need to change the `pre_release_tag_re`, the default looks like it should work: `(?P<pre_release>\.v?\d+(?:[ab]|rc)+\d*)$` but I've had issues with it not collapsing pre-releases in the past. But we can fix this post release, because I'm not sure there is a way to check this until we have the tag.